.github/workflows/dcaf-build-workflow.yml

```diff
@@ -9,9 +9,9 @@ jobs:
       - uses: actions/checkout@v1
         with:
           fetch-depth: 1
-      - uses: actions/setup-ruby@v1
+      - uses: actions/setup-ruby@v1.1.0
```

Comment:
Since github actions has really shaped up to be its own horrible beast, can we revert these two changes?

Reply:
Absolutely

Reply:
In: d86bb5e